recipes/ltr_finder_parallel/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "LTR_FINDER_parallel" %}
+{% set version = "1.1" %}
+{% set sha256 = "5d1fed2a3a38c0927ccd7fc6cc38ba13ce293c5b1805299c999c5d52d085a1b9" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/oushujun/LTR_FINDER_parallel/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: generic
+  run_exports:
+    - {{ pin_subpackage('ltr_finder_parallel', max_pin='x') }}
+
+requirements:
+  run:
+    - ltr_finder ==1.07
```

Comment:
perl is a dependency, but i guess that's already covered by ltr_finder itself requiring perl

Reply:
Yes, that's true. `perl` is provided by  ltr_finder itself.